<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-GFUJ2UT" failures="1" tests="1" name="runner.TestRunner" time="51.153" errors="0" timestamp="2024-01-17T18:33:57 IST" skipped="0">
  <testcase classname="runner.TestRunner" name="runScenario" time="51.153">
    <failure message="expected [Oops! Payment not confirmed!] but found [https://uat.fnp.ae/control/checkout-page-rj#/checkout-page]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Oops! Payment not confirmed!] but found [https://uat.fnp.ae/control/checkout-page-rj#/checkout-page]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at stepdefination.FnpLogin.verify_that_user_land_on_pnc_page_as_a(FnpLogin.java:884)
at âœ½.verify that user land on PNC page as a "Oops! Payment not confirmed!"(file:///C:/Users/Aman%20Tanmay%20Varshney/Downloads/FNP_PROJECT%20-%20Aman/src/test/resources/features/fnp.feature:241)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- runner.TestRunner -->
